### ISSUE TITLE
Adding missing translations

### ISSUE THREAD

**Issue by rodzy** [HospitalRun/hospitalrun-frontend#2104]
## 🚀 Feature Proposal

To add some of the missing translations on the script files, can I start working on them or you guys planning on doing it a different way?

## Motivation

I've seen some traction for this repository on some Facebook front-end groups and they were talking about the missing languages , so I've been studying the code and would like to collaborate

**Comment by rodzy** [comment]
Hi @tehKapa. I'm catching up with the Crowdin documentation right now.
As soon I have it done I notify you guys on Slack

**Comment by tehKapa** [comment]
The main issue is finding a way to implement a translation template to the current code structure.

**Comment by marcosvega91** [comment]
The best way in this case is to use JSON file instead of ts files for translations

**Comment by tehKapa** [comment]
And use this https://www.npmjs.com/package/react-i18next for import json?

**Comment by marcosvega91** [comment]
Yes, I think is simplier in this way rather than create all ts files. What do you think ?

**Comment by tehKapa** [comment]
You're probably right (I'm not a developer, I'm in charge of management and CI/CD). 

Btw, I leave here a reference link https://github.com/i18next/react-i18next/issues/472

**Comment by marcosvega91** [comment]
It is not a problem :). About the implementation I think that with .json is simpler than with .ts.
As i suggested here #1919 [Docusaurus](https://github.com/facebook/docusaurus/blob/master/docs/guides-translation.md) is using this service. @rodzy I think that you can take a look their implementation.

**Comment by rodzy** [comment]
I was messing with the .ts files yesterday and it's definitely challenging, because Crowdin only uses string recognition and some times picks the type properties as strings that in mind changes the flow results on not displaying any data on the end application.
I might try using .json , thx for the suggestion on Docusaurus @marcosvega91, I'm definitely checking their implementation.

**Comment by comradekingu** [comment]
@tehKapa Stay far away from Crowdin, see https://github.com/jitsi/jitsi-meet/issues/5056#issuecomment-612239691
Use Weblate instead :)

**Comment by tehKapa** [comment]
@comradekingu thanks for the advice!

**Comment by tehKapa** [comment]
Here we are! Weblate support has lightning-fast activated a Libre account https://hosted.weblate.org/projects/hospitalrun/

@rodzy @marcosvega91 unfortunately it was better to change service, I hope that the work done on Crowdin can be reused

**Comment by comradekingu** [comment]
@tehKapa Better idea that discovering the quality isn't good.
I can help populate it if you add https://hosted.weblate.org/user/kingu/ as an admin :)
Only started helping projects this way, but got two projects up and running in as many days.

**Comment by fox1t** [comment]
Thanks, @comradekingu for pointing this out. As a project that will be used by final users, we need to pay more attention to who we rely upon!

**Comment by comradekingu** [comment]
I was going to say I couldn't agree more, but then I remembered I proposed being an admin ;)

**Comment by tehKapa** [comment]
@comradekingu I've added you to Weblate project

**Comment by comradekingu** [comment]
So from what I can tell .ts Qt Linguist files are supposed to be different from the ones found in this repo. No luck adding those.

Start tag expected, '<' not found, line 1, column 1 (<string>, line 1) src/shared/locales/de/translations/actions/index.ts: Start tag expected, '<' not found, line 1, column 1 (<string>, line 1) src/shared/locales/enUs/translations/actions/index.ts: Start tag expected, '<' not found, line 1, column 1 (<string>, line 1) src/shared/locales/es/translations/actions/index.ts: Start tag expected, '<' not found, line 1, column 1 (<string>, line 1) src/shared/locales/fr/translations/actions/index.ts: Start tag expected, '<' not found, line 1, column 1 (<string>, line 1) src/shared/locales/id/translations/actions/index.ts: Start tag expected, '<' not found, line 1, column 1 (<string>, line 1) src/shared/locales/it/translations/actions/index.ts: Start tag expected, '<' not found, line 1, column 1 (<string>, line 1) src/shared/locales/ja/translations/actions/index.ts: Start tag expected, '<' not found, line 1, column 1 (<string>, line 1) src/shared/locales/ptBr/translations/actions/index.ts: Start tag expected, '<' not found, line 1, column 1 (<string>, line 1) src/shared/locales/ru/translations/actions/index.ts: Start tag expected, '<' not found, line 1, column 1 (<string>, line 1) src/shared/locales/zhCN/translations/actions/index.ts: Start tag expected, '<' not found, line 1, column 1 (<string>, line 1)

**Comment by tehKapa** [comment]
what are the steps you have done?

**Comment by comradekingu** [comment]
I added a component with settings I think would otherwise have worked. Tried both with wildcards for component and language, and adding just one component at a time. I can post the settings if you like.

**Comment by tehKapa** [comment]
yep, it would be useful

**Comment by client-side96** [comment]
Hi guys, I'm new to this project and to be honest also new to "contributing on GitHub" in general. I saw that there are missing some German translations and wanted to ask if I could help you adding them ?

**Comment by tehKapa** [comment]
Hi @client-side96. We wanted to use Weblate to make translation easier, but everything remained on standby. 
Actually, we manually translate files with the various labels, like this PR: https://github.com/HospitalRun/hospitalrun-frontend/pull/2661
